### PR TITLE
Make CL2 accept multiple master ips.

### PR DESCRIPTION
**Pull request by mm4tt**
This change should be no-op, it only changes the CL2 api (flags/envs) to accept multiple values as the `master ip` - both internal and external. 
The api change should be backward compatible, i.e. if flag/env was set to a single master-ip it will continue to work in exactly the same way.

Actual changes supporting multi-master clusters will come in separate PRs.

### CODE REVIEW

File: clusterloader2/cmd/clusterloader.go [hunk 1]
```diff
@@ -57,8 +57,8 @@ func initClusterFlags() {
 	flags.IntEnvVar(&clusterLoaderConfig.ClusterConfig.Nodes, "nodes", "NUM_NODES", 0, "number of nodes")
 	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.Provider, "provider", "PROVIDER", "", "Cluster provider")
 	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.MasterName, "mastername", "MASTER_NAME", "", "Name of the masternode")
-	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.MasterIP, "masterip", "MASTER_IP", "", "Hostname/IP of the masternode")
-	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.MasterInternalIP, "master-internal-ip", "MASTER_INTERNAL_IP", "", "Cluster internal/private IP of the master vm")
+	flags.StringSliceEnvVar(&clusterLoaderConfig.ClusterConfig.MasterIPs, "masterip", "MASTER_IP", nil /*defaultValue*/, "Hostname/IP of the master node, supports multiple values when separated by commas")
```
Comment (wojtek-t) [inline clusterloader2/cmd/clusterloader.go]: I understand that this needs to be done in steps, but we should also change the flag name and env-var name to plural.
At least a TODO for that please.

File: clusterloader2/pkg/flags/types.go [hunk 2]
```diff
@@ -17,9 +17,9 @@ limitations under the License.
 package flags
 
 import (
-	"strconv"
-
 	"github.com/spf13/pflag"
+	"strconv"
```
Comment (wojtek-t) [inline clusterloader2/pkg/flags/types.go]: Please revert - the convention is that the first group of important are builtin go libraries, exactly how it was before.

File: clusterloader2/pkg/prometheus/prometheus.go [hunk 3]
```diff
@@ -228,9 +228,9 @@ func dumpAdditionalLogsOnPrometheusSetupFailure(k8sClient kubernetes.Interface)
 }
 
 func getMasterIp(clusterConfig config.ClusterConfig) (string, error) {
-	if clusterConfig.MasterInternalIP != "" {
-		klog.Infof("Using internal master ip (%s) to monitor master's components", clusterConfig.MasterInternalIP)
-		return clusterConfig.MasterInternalIP, nil
+	if clusterConfig.GetMasterIp() != "" {
```
Comment (wojtek-t) [inline clusterloader2/pkg/prometheus/prometheus.go]: internal

Comment (mm4tt) [reply clusterloader2/pkg/prometheus/prometheus.go]: Done.

File: clusterloader2/pkg/flags/flags.go [hunk 4]
```diff
@@ -65,6 +66,18 @@ func StringArrayVar(s *[]string, flagName string, defaultValue []string, descrip
 	pflag.StringArrayVar(s, flagName, defaultValue, description)
 }
 
+// StringSliceEnvVar creates a string slice flag with the given parameters.
```
Comment (wojtek-t) [inline clusterloader2/pkg/flags/flags.go]: BTW - this support for string-slice flags, is orthogonal to the overal change, so if you could split that into separate PR, we could merge it now :)

Comment (mm4tt) [reply clusterloader2/pkg/flags/flags.go]: Was done some time ago :)

File: clusterloader2/pkg/flags/types.go (hunk 2 above)
Comment (mm4tt) [reply clusterloader2/pkg/flags/types.go]: Outdated

File: clusterloader2/cmd/clusterloader.go (hunk 1 above)
Comment (mm4tt) [reply clusterloader2/cmd/clusterloader.go]: Added TODO.

**Comment by mm4tt** [comment]
@mborsz, @wojtek-t, PTAL

**Comment by wojtek-t** [comment]
lgtm
